src/core/QCheck2.ml

```diff
@@ -420,15 +435,15 @@ module Gen = struct
   let int : int t =
     bool >>= fun b ->
     if b
-    then pint ~origin:0 >|= (fun n -> - n - 1)
-    else pint ~origin:0
+    then nat_origin 0 >|= (fun n -> - n - 1)
+    else nat_origin 0
 
   let int_bound (n : int) : int t =
     if n < 0 then invalid_arg "Gen.int_bound";
     fun st ->
       if n <= (1 lsl 30) - 2
       then Tree.make_primitive (fun a () -> Shrink.int_towards 0 a ()) (RS.int st (n + 1))
-      else Tree.map (fun r -> r mod (n + 1)) (pint st)
+      else Tree.map (fun r -> r mod (n + 1)) (nat st)
```

Comment:
I think suspect this renaming introduces a bug: the else branch handles `n` bigger than 30-bit,
which will then need the old `pint` (now: `nat_origin`) to stitch together 3 calls.
Using plain `nat` outputs max 10.000 though. A statistics test should reveal the change in distribution!
This would be a good opportunity to add it 😀